server/controllers/manage/placementController.test.ts

```diff
@@ -44,8 +86,76 @@ describe('placementController', () => {
           pageHeading: '16 Nov 2024 to 26 Mar 2025',
           user,
           backLink: null,
+          activeTab: 'placement',
+        }),
+      )
+      expect(premisesService.getPlacement).toHaveBeenCalledWith({ token, placementId: placement.id, premisesId })
+      expect(applicationService.findApplication).not.toHaveBeenCalled()
```

Comment:
Would be nice to assert _all_ the endpoints that have _not_ been called!

Reply:
OK